tensorflow/core/kernels/mkl_matmul_op.cc

```diff
@@ -150,11 +155,25 @@ class MklMatMulOp : public OpKernel {
     // 1.0 and 0.0 respectively.
     const float alpha = 1.0f;
     const float beta = 0.0f;
+#if defined(DO_NOT_USE_ML)
+    const char* const ftrans[]   = { "N", "T", "C"};
+    int index_transa = transa? 1 : 0 ;
+    int index_transb = transb? 1 : 0 ;
+    VLOG(2) << "MKL DNN SGEMM called";
+    //Using the fortran api of mkldnn
```

Comment:
Is this Fortran API or C++ API expecting column major layout? Please make clear.